docs/source/api_reference/image_video_overlay.rst

```diff
@@ -44,6 +44,6 @@ Attributes
 ===========    ========================   ===
 Attribute      Default Value              Doc
 ===========    ========================   ===
-url            ""                         An http url to the footage or a relative path to a local file (image/video). Note that absolute local paths are not supported. 
+url            ""                         An http url to the footage or a relative path to a local file (image/video, e.g. ‘image.png’ or ‘files/working_dir/image.png’ in JupyterLab). Note that absolute local paths are not supported. 
```

Comment:
What is different in JupyterLab, the `files/` prefix?

Reply:
Yes, otherwise it does not work.

Reply:
Maybe we should make it clearer, like: "Note that in JupyterLab the path must be prefixed with `files/`, e.g. `files/image.png`".

Reply:
Also important is the working_dir so you have to add "files/my_working_dir/", this is a bit strange and it is not mentioned anywhere.

Reply:
It is mentioned there in the documentation: https://ipyleaflet.readthedocs.io/en/latest/api_reference/local_tile_layer.html#example

Reply:
In Jupyter Notebook the path is relative to the Notebook. In JupyterLab it's relative to the server.

Reply:
> Also important is the current working folder name so you have to add "files/my_working_dir/", this is a bit strange and it is not mentioned anywhere.

Yes but this is not specific to JupyterLab, so we should make it consistent with the notebook.

Reply:
> In Jupyter Notebook the path is relative to the Notebook. In JupyterLab it's relative to the server.

Thanks, then this should also be mentioned.